views/get-involved.jade

```diff
@@ -52,13 +52,17 @@ block content
     h2 Project contributions
 
     h3 Bug reports
-    p If you notice something that does not work as it should, and it&rsquo;s not listed on know issues. Please leave a bug report.
+    p If you notice something that does not work as it should, and it&rsquo;s not listed on known issues. Please leave a bug report.
     p
-      a(href='https://github.com/ubports/ubuntu-touch-for-oneplus-one/issues') Report a bug
+      a(href='https://bugs.launchpad.net/ubports-android') Report a bug
     h3 Code
-    p If you want to help developing and porting Ubuntu touch why not help us. If you have a device that you want to port and maintain, please drop us a email to me@mariogrip.com  or pm mariogrip on the forums.
+    p If you want to help developing and porting Ubuntu Touch why not help us. If you have a device that you want to port and maintain, please drop us a email to me@mariogrip.com  or pm mariogrip on the forums.
     p
       a(href='http://github.com/ubports') Fork/patch our source code
-
+    h3 IRC (Internet Relay Chat)
+    p If you want to chat with the developers in real time you can join us in IRC though this Kiwi IRC Embedded Chat!
+    
+    <iframe src="https://kiwiirc.com/client/irc.freenode.net/?nick=UBport-use|?#ubuntu-touch, #ubports" style="border:0; width:100%; height:450px;"></iframe>
```

Comment:
HTML won't work, you need to translate it to jade. this does the translation pretty easy http://html2jade.aaron-powell.com/